coffee_core/src/config.rs

```diff
@@ -52,6 +52,7 @@ impl CoffeeConf {
         info!("creating coffee home at {def_path}");
         check_dir_or_make_if_missing(format!("{def_path}/bitcoin")).await?;
         check_dir_or_make_if_missing(format!("{def_path}/testnet")).await?;
+        check_dir_or_make_if_missing(format!("{def_path}/repositories")).await?;
```

Comment:
why we need this here?

Reply:
If we don't create it when configuring at first, the user will get an error if he tries to use any command before adding a remote repository.
For example, if he tries to list the remotes, he will get an error saying that the `repositories` folder wasn't found.

I think it makes more sense to create it beforehand and save the error for the case that the user actually messes it up somehow by accident.

Reply:
yes it does! thanks

could you add this explanation inside the commit body? so we know why you change this code